servers/xr/xr_server.cpp

```diff
@@ -29,9 +29,10 @@
 /**************************************************************************/
 
 #include "xr_server.h"
+
 #include "core/config/project_settings.h"
-#include "xr/xr_interface.h"
-#include "xr/xr_positional_tracker.h"
+#include "xr_interface.h"
+#include "xr_positional_tracker.h"
 #include "xr_server.compat.inc"
```

Comment:
The convention here would be:
```
#include "xr_server.h"
#include "xr_server.compat.inc"

#include "core/config/project_settings.h"
#include "servers/xr/xr_interface.h"
#include "servers/xr/xr_positional_tracker.h"
```
- `compat` file next to main header
- all other includes are "absolute" (from root of the repo)